<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>693</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>5700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <property name="maximumSize">
          <size>
           <width>532</width>
           <height>605</height>
          </size>
         </property>
         <attribute name="title">
          <string>Common</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>580</y>
            <width>374</width>
            <height>25</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Your message:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="messageLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="sendMessageButton">
             <property name="text">
              <string>Send to all</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="privateMessageSendButton">
             <property name="text">
              <string>Send private</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Common chat</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>10</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Private chat</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>531</width>
            <height>541</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTextBrowser" name="commonChatBrowser"/>
          <widget class="QTextBrowser" name="privateChatBrowser"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <property name="maximumSize">
          <size>
           <width>532</width>
           <height>605</height>
          </size>
         </property>
         <attribute name="title">
          <string>UserPrivateChat</string>
         </attribute>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Private messge To</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>10</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Privatemessage From</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>0</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>531</width>
            <height>541</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTextBrowser" name="commonChatBrowser_2"/>
          <widget class="QTextBrowser" name="privateChatBrowser_2"/>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>0</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>SetUserName</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>580</y>
            <width>521</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>WARNING: Set UserName to filter messages</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <property name="maximumSize">
          <size>
           <width>532</width>
           <height>605</height>
          </size>
         </property>
         <attribute name="title">
          <string>Additional</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>520</y>
            <width>511</width>
            <height>25</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Chat User to BANN</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Press This BUTTON ---&gt;&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>BANN</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>511</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QTextBrowser" name="privateChatBrowser_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>511</width>
            <height>391</height>
           </rect>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>511</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>511</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="text">
           <string>Here You can watch ALL Private messages from anyone to anyone</string>
          </property>
         </widget>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>500</y>
            <width>511</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_menu">
    <property name="title">
     <string>Main menu</string>
    </property>
    <addaction name="actionOpen_another_cliend"/>
    <addaction name="separator"/>
    <addaction name="actionCloseClient"/>
   </widget>
   <addaction name="menuMain_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_another_cliend">
   <property name="text">
    <string>Open another client</string>
   </property>
  </action>
  <action name="actionCloseClient">
   <property name="text">
    <string>Close this client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
